Location fix

marcvaneijk opened the pull request:
Changed resource location from parameter to resourceGroup().location.
Deleted location parameter from deployment template and parameter file.

singhkay commented:
@marcvaneijk  error `AssertionError: symantec-extension-windows-vm/metadata.json - #/itemDisplayName:Failed "maxLength" criteria`

marcvaneijk commented:
@singhkay 
- Fixed max length itemDisplayName
- storageAccountName from param to var
- replaced disk endpoints to dynamic
- removed storageAccountName from param file
- updated param file with placeholders
